src/index.json

```diff
@@ -2181,6 +2181,49 @@
                 "sha256Digest": "42c6500e5587184026a398147b0a7b1a9cb7e59642de9b932dfcbe1a90f14697"
             }
         ],
+        "vmware-cs": [
+            {
+                "downloadUrl": "https://github.com/Azure/az-vmware-cli/releases/download/v0.1.0/vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "filename": "vmware_cs-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.1.0",
```

Comment:
you will have to switch sequence of these 2 lines, minCliExtension on the top of maxCliExtension.
Don't ask me why.... but it disrupts validation

Reply:
Done.